src/exploreLondon/CommunityPartnerCard.styles.js

```diff
@@ -1,20 +1,22 @@
 import styled from 'styled-components'
-import Img from 'gatsby-image'
+import { compose, layout, space } from 'styled-system'
+import Image from 'gatsby-image/withIEPolyfill'
 import theme from '../theme/theme'
 import { media } from '../theme/media'
+import { colors } from '../theme/colors'
 import { NewsCategory } from '../components/newsCategory'
+import { Card } from '../components/card'
 
-export const Card = styled.div`
-  height: 100%;
-  border-radius: 4px;
-  overflow: hidden;
-  background-color: ${theme.colors.white};
+export const StyledImage = styled(Image)`
```

Comment:
The rendered image doesn't match the images on the current site ratio/dimensions.

Reply:
Reverted 👍 .

Reply:
I left the same category filter as the one on https://prideinlondon.org/fifty-two but I can revert if needed.